### PR TITLE
Upgraded to oclif v2

### PR DESCRIPTION
**Pull request by chrbrt**
+upgraded oclif to v2
+includes everything in the [compatibility matrix](https://oclif.io/blog/2022/01/12/announcing-oclif-v2)
˜fixed all issues relating to upgrades

### CODE REVIEW

File: bin/dev [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+const oclif = require('@oclif/core');
+
+const path = require('path');
+const project = path.join(__dirname, '..', 'tsconfig.json');
+
+// In dev mode -> use ts-node and dev plugins
```
Comment (Templum) [inline bin/dev:8]: The whole thing is in dev mode all the time
```suggestion
```

File: src/commands/create/index.ts [hunk 2]
```diff
@@ -8,13 +8,13 @@ export class Create extends Command {
...
   public static flags = {
-    help: flags.help({
+    help: Flags.help({
       char: 'h',
       description: 'Show help for the create command',
     }),
   };
 
-  public static description = 'Create functions and schedules locally';
+  public static description = 'Create functions locally nd schedules remotely';
```
Comment (Templum) [inline src/commands/create/index.ts:17]:
```suggestion
  public static description = 'Create functions locally and schedules remotely';
```

File: supertest/functions/biggestboi/config.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "biggestboi",
```
Comment (Templum) [inline supertest/functions/biggestboi/config.json:2]: Is this supposed to be here at all ? 

I call Sus!

**Review by Templum — CHANGES_REQUESTED** [review]

File: supertest/functions/biggestboi/config.json (hunk 3 above)
Comment (chrbrt) [reply supertest/functions/biggestboi/config.json]: 😅

**Review by Templum — DISMISSED** [review]

**Pull request merged**